sortable-list.html

```diff
@@ -193,6 +199,34 @@
         }
       },
 
+      handleGroupMove: function(fromGroup, toGroup, removeIndex, addIndex) {
```

Comment:
private function should have  an underscore, and let's put a nice comment on this guy.